includes/functions.php

```diff
@@ -152,10 +152,20 @@ function tsml_calculate_attendance_option($types, $approximate)
 
 //called by register_activation_hook in 12-step-meeting-list.php
 //hands off to tsml_custom_post_types
-function tsml_change_activation_state()
+function tsml_plugin_activation()
 {
 	tsml_custom_post_types();
 	flush_rewrite_rules();
+    add_option( 'tsml_plugin_new_activation', true );
+    update_option( 'tsml_plugin_new_activation', true );
```

Comment:
Maybe I'm doing this wrong, but if an option doesn't exist, you need to add it, and if it does you have to update it. Easier to do both then check and do one or the other.